core/common/src/AnalysisStyle.ts

```diff
@@ -13,6 +13,7 @@ import { Gradient } from "./Gradient";
 /** JSON representation of an [[AnalysisStyleDisplacement]].
  * @see [[AnalysisStyleProps.displacement]].
  * @public
+ * @extensionApi
```

Comment:
What do we think of using the `@extension` rather than `@extensionApi`?

Reply:
Fine with me. I prefer one word....maybe @extensions plural is more meaningful (the object itself is not an "extension" but rather able to be used by "extensions")? Good either way